### PR TITLE
Changed default logging to WARN for doctests

### PR DESCRIPTION
**Pull request by eracah**
# What does this PR do?

Changes to logging level to WARN for doctests. This reduces the verbosity of doctests and makes it easier to see your error.

# What issue(s) does this change relate to?

CO-1285

# Before submitting
- [x] Have you read the [contributor guidelines](https://github.com/mosaicml/composer/blob/dev/CONTRIBUTING.md)?
- [ ] Is this change a documentation change or typo fix? If so, skip the rest of this checklist.
- [x] Was this change discussed/approved in a GitHub issue first? It is much more likely to be merged if so.
- [ ] Did you update any related docs and document your change?
- [ ] Did you update any related tests and add any new tests related to your change? (see [testing](https://github.com/mosaicml/composer/blob/dev/CONTRIBUTING.md#running-tests))
- [x] Did you run the tests locally to make sure they pass?
- [x] Did you run `pre-commit` on your change? (see the `pre-commit` section of [prerequisites](https://github.com/mosaicml/composer/blob/dev/CONTRIBUTING.md#prerequisites))

### CODE REVIEW

**Comment by karan6181** [comment]
Changes looks good to me. I have one question though, is it possible to customize logging based on the command-line parameter? for example, if user pass `-v` flag then the logging defaults to `DEBUG` and if user doesn't pass `-v` flag then it remain at `WARNING` level.

File: docs/source/doctest_fixtures.py [hunk 1]
```diff
@@ -10,6 +10,9 @@
 so all imports and variables are available in any doctest.
 The output of this setup script does not show up in the documentation.
 """
+import logging
+
+logging.basicConfig(level=logging.WARN)
```
Comment (dakinggg) [inline docs/source/doctest_fixtures.py:15]: wait isn't the python default warn? is somewhere else setting it to info?

Comment (eracah) [reply docs/source/doctest_fixtures.py:15]: Not sure, but when you run doctests, it prints INFO statements, which clutter up stdout and stderr.

**Comment by eracah** [comment]
> Changes looks good to me. I have one question though, is it possible to customize logging based on the command-line parameter? for example, if user pass `-v` flag then the logging defaults to `DEBUG` and if user doesn't pass `-v` flag then it remain at `WARNING` level.

I don't think so unless sphinx lets you add custom command line arguments.

**Pull request merged**